specification/resources/resource-manager/Microsoft.Authorization/2016-12-01/policyDefinitions.json

```diff
@@ -256,13 +279,6 @@
         "description": "Gets all the policy definitions for a subscription.",
         "parameters": [
           {
-            "name": "$filter",
```

Comment:
So this was not supported and here by mistake?

Reply:
Yes, apparently, only policy assignment list supports filter. Not policy definitions and set definitions

Reply:
ok, this was exposed in the CLI, I create an issue for that

Reply:
https://github.com/Azure/azure-cli/issues/4603